### DISCUSSION TITLE
easypanel for arm base vps

### DISCUSSION THREAD

**Discussion by mc-stephen** [easypanel-io/community#70] [General]
I tried to install Easypanel on my oracle server but it didn't work cus am using an arm64/v8 base VPS, so I want to know if Easypanel is available for that?

thanks in advance.

**Comment by deiucanta** [comment]
For now, we don't support ARM but I'm considering it for the future. There are two parts to this change.

- make Easypanel work on ARM
- make all the services work on ARM (databases and templates)

For the templates, the effort is out of the scope. Some templates will work on ARM, some will not. But for the other services (databases) I think they all support ARM already.

Are you using Oracle because of their generous free plan?

**Comment by mc-stephen** [comment]
Okay, I can't wait for it to support ARM,

and yes I am using oracle cloud VPS for this, and truly their always free plan is very welcoming.
I can't wait to gain some income from it, I will really love to purchase from them later in the future.

anyways thanks for answering me, I hope it won't take long ARM support to come out :)

**Reply by deiucanta** [reply]
Ok, I added experimental ARM support :) 

If you want to try it now, you should install the canary version. Also, I highly recommend you add a swap of 4GB.

1. Add swap
    - https://www.digitalocean.com/community/tutorials/how-to-add-swap-space-on-ubuntu-22-04

2. Install docker

    ```bash
    curl -sSL https://get.docker.com | sh
    ```

3. Install Easypanel Canary

    ```bash
    docker run --rm -it \
      -v /etc/easypanel:/etc/easypanel \
      -v /var/run/docker.sock:/var/run/docker.sock:ro \
      -e RELEASE_TAG=canary \
      easypanel/easypanel:canary setup
    ```

**Reply by deiucanta** [reply]
Don't forget to configure the firewall on Oracle Cloud to allow inbound traffic.

**Reply by mc-stephen** [reply]
sorry for the late reply (i have been busy), 
Thanks i will try it out, when i do i will give my feedback on how it went.
Tho i don't fully understand how Easypanel works, i saw a new saas called caprover so amma compare it with Cyberpanel and try both out, and then i will decide which is best for me.
anyways thanks to you guys and the open-source community for making Easypanel possible today :)

**Comment by mc-stephen** [comment]
@deiucanta
nice Easypanel looks good, I just installed it, but I need to ask how will install Appwrite on it,

**Reply by deiucanta** [reply]
Appwrite is a bit tricky to install since it has a lot of services. You should follow this `docker-compose.yml` if you want to install it. https://github.com/appwrite/appwrite/blob/master/docker-compose.yml
